### DISCUSSION TITLE
Hero 11 Scene Motion Detection sensitivity

### DISCUSSION THREAD

**Discussion by Bocio** [gopro/labs#813] [General]
I'm trying to use this function to film closely some huge herons on their trees but no luck so far. 
The scene has several trees and moving leaves. I set the mask to 20 seconds and I cannot use an image mask.
It seems to be no linearity in the sensitivity settings. 
Sensitivity = 2 and I get nearly continuous recordings. Even a sudden change of luminosity (sun and clouds change the leaves color) triggers the camera. 
Sensitivity = 1 and I miss the heron arriving slowly in front of the camera.

Any suggestion?

**Comment by dnewman-gpsw** [comment]
You like have moment from leaves in the shot. Start your capture at a windy moment if you can and set the Motion Mask to 10 or so seconds.  This will determine which parts of the frame to ignore as having false triggers.

**Reply by Bocio** [reply]
I have to catch a windy moment because I already set the mask to 20".
I will try, thanks
